<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dsdemoClass</class>
 <widget class="QMainWindow" name="dsdemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Depthstream Demo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/Scratch/stereo-camera_318-100805.jpg</normaloff>D:/Scratch/stereo-camera_318-100805.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="depthstream_label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>D:/Dropbox/Documents/Thesis/label.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="depthmap_demo_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Depth Map Demo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tracking_demo_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tracking Demo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pointcloud_demo_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PointCloud Demo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dsdemo.qrc"/>
 </resources>
 <connections/>
</ui>
